modules/microshift-greenboot-included-health-checks.adoc

```diff
@@ -0,0 +1,21 @@
+//Module included in the following assemblies:
+//
+//* microshift_running-applications/microshift-greenboot-workload-scripts.adoc
+
+:_content-type: CONCEPT
+[id="microshift-greenboot-included-health-checks_{context}"]
+= Included greenboot health checks
+
+Health check scripts are available in `/usr/lib/greenboot/check`, a read-only directory in RPM-OSTree systems. The following health checks are included with `greenboot-default-health-checks`.
+
+* Check if repository URLs are still DNS solvable:
++
+This script is under `/usr/lib/greenboot/check/required.d/01_repository_dns_check.sh` and ensures that DNS queries to repository URLs are still available.
+
+* Check if update platforms are still reachable:
++
+This script is under `/usr/lib/greenboot/check/wanted.d/01_update_platform_check.sh` and tries to connect and get a 2XX or 3XX HTTP code from the update platforms defined in `/etc/ostree/remotes.d`.
+
+* Check if the current boot has been triggered by hardware watchdog:
++
+This script is under `/usr/lib/greenboot/check/required.d/02_watchdog.sh` and checks whether the current boot has been watchdog-triggered or not. If it is, but the reboot has occurred after a certain grace period (default of 24 hours, configurable via `GREENBOOT_WATCHDOG_GRACE_PERIOD=number_of_hours` in `/etc/greenboot/greenboot.conf`), greenboot does not mark the current boot as red and does not rollback to the previous deployment. If it has occurred within the grace period, the current boot will be marked as red, but greenboot does not rollback to the previous deployment. It is enabled by default. It can be disabled by modifying `GREENBOOT_WATCHDOG_CHECK_ENABLED` in `/etc/greenboot/greenboot.conf to false`.
```

Comment:
I'm not sure what `It` refers to in `It can be disabled by modifying..` Part of the confusion for me is the back-to-back uses of it. Maybe define some of them. I also think this could be another good spot for a [NOTE] to pullout important information that is currently positioned in an interrupting phrase (default of 24 hours, configurable via...). Pulling that phrase out also helps the flow of the prose here. See what you think.